geoips/commandline/geoips_tree.py

```diff
@@ -144,15 +165,29 @@ def print_tree(self, parser, colored=False, level=0, max_depth=2, short_name=Fal
             # The case above would result in 'geoips config install'
             cmd_name = " ".join(split_cmd[3:5] + [split_cmd[-1]])[1:]
 
+        if (
+            cmd_name == "geoips describe sectors" and self.describe_sectors_outputted
+        ) or (cmd_name == "geoips list sectors" and self.list_sectors_outputted):
+            # Due to how we've structured aliases, these commands need to be manually
+            # checked that they've only ran once. This is caused by 'geoips test sector'
+            # command, where 'sector is the actual name of a command so we can't include
+            # it in the 'cmd_aliases' property of this class.
+            return
+
         if short_name:
             # Just grab the exact name of the command.
             # Ie. 'geoips' or 'config' or 'install'
             cmd_txt = f"{split_cmd[-1]}"
         else:
             cmd_txt = cmd_name
 
-        url = f"{self.cmd_line_url}#{cmd_name.replace(' ', '-')}"
-        hyperlink = self.link(url, cmd_txt)
+        # Commenting these portions out until we can get the documentation build and
+        # make this easily testable. The introduction of Class Factories for commands
+        # has made this much harder to implement.
+
+        # url = f"{self.cmd_line_url}#{cmd_name.replace(' ', '-')}"
+        # hyperlink = self.link(url, cmd_txt)
+        hyperlink = cmd_txt
 
         if colored:
```

Comment:
We should probably not use this word in code!

First dictionary entry: 

Colored is a racial descriptor historically used in the United States during the Jim Crow Era to refer to an African American. In many places, it may be considered a slur.

Reply:
Sure; I'll fixed that!

Reply:
Fixed.

Reply:
Thanks! :)